assets/course-theme/adminbar-layout.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Track how much space the WordPress admin bar takes up at the top of the screen.
+ * Updates a CSS variable with the value.
+ */
+const trackAdminbarOffset = () => {
```

Comment:
For logged in admin users of the site, the Wordpress admin bar is at the top of the page. This is fixed for desktop, fixed but larger on smaller screens, and then static (moves with the scrollable content) on small screens. This script is for adjusting spacing for the sidebar when it's static.